lib/Listener/MessageCacheUpdaterListener.php

```diff
@@ -59,11 +59,15 @@ public function handle(Event $event): void {
 			$message->setFlag($event->getFlag(), $event->isSet());
 
 			$this->mapper->update($message);
-		} elseif ($event instanceof MessageDeletedEvent) {
-			$this->mapper->deleteByUid(
-				$event->getMailbox(),
-				$event->getMessageId()
-			);
+		} elseif ($event instanceof MessagesDeletedEvent) {
+			$mailbox = $event->getMailbox();
+			$uids = $event->getMessageIds();
+			foreach ($event->getMessageIds() as $uid) {
```

Comment:
```suggestion
			foreach ($event->getMessageUids() as $uid) {
```

Reply:
This is the only usage of `getMessageUids()`.

Reply:
yes indeed. Done